yuniql-platforms/oracle/Yuniql.Oracle.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <ApplicationIcon />
+    <OutputType>Library</OutputType>
+    <StartupObject></StartupObject>
+    <AssemblyName>Yuniql.Oracle</AssemblyName>
+    <RootNamespace>Yuniql.Oracle</RootNamespace>
+    <Version>0.0.1</Version>
+    <Authors>Rodel E. Dagumampan</Authors>
```

Comment:
You are welcome replace this name here :)

Reply:
Nah, it looks good this way :-). Most of the stuff is copied so it good to credit them :-P